juniper_official/Solutions/EVPN-VXLAN/vxlan-vport.rule

```diff
@@ -5,96 +5,97 @@ healthbot {
+                argument used {
+                    mandatory;
+                }
             }
-        }
-        sensor vport-information {
+            sensor vport-information {
                 synopsis "vport netconf based iAgent sensor definition";
                 description "iAgent sensor defined in yml file using netconf rpc/cli command and defined fields using pyez tables&views to collect data from network device";
-            iAgent {
-                file vport.yml;
-                table VirtualVportTable;
-                frequency 60s;
+                iAgent {
+                    file vport.yml;
+                    table VirtualVportTable;
+                    frequency 60s;
+                }
             }
-        }
-        field vport-count {
-            formula {
-                user-defined-function {
-                    function-name difference;
-                    argument subtract_from "$vport_count";
-                    argument subtract 1;
+            field vport-count {
+                formula {
+                    user-defined-function {
+                        function-name difference;
+                        argument subtract_from "$vport_count";
+                        argument subtract 1;
+                    }
                 }
+                type integer;
             }
-            type integer;
-        }
-    field vport-percentage-used {
-            formula {
-                user-defined-function {
-                    function-name used-percentage;
-                    argument used "$vport-count";
-                    argument total "{{vport_max_count}}";
+            field vport-percentage-used {
+                formula {
+                    user-defined-function {
+                        function-name used-percentage;
+                        argument used "$vport-count";
+                        argument total "{{vport_max_count}}";
+                    }
                 }
+                type integer;
             }
-            type integer;
-        }
-        trigger vport-usage {
+            trigger vport-usage {
                 synopsis "vport kpi";
                 description "Sets health based on vport used percentage.";
-            term below-threshold {
-                when {
-                    less-than "$vport-percentage-used" 60;
-                }
-                then {
-                    status {
-                        color green;
-                        message " vport count is $vport-count which is  $vport-percentage-used  %of max";
+                frequency 90s;
+                term below-threshold {
+                    when {
+                        less-than "$vport-percentage-used" 60;
+                    }
+                    then {
+                        status {
+                            color green;
+                            message " vport count is $vport-count which is  $vport-percentage-used  %of max";
```

Comment:
1. Remove space in begin of message
2. s/" vport count is $vport-count which is  $vport-percentage-used  %of max";/"Vport count($vport-count) and used percentage($vport-percentage-used  %) is normal";/g